dss/util/auth/auth0.py

```diff
@@ -22,7 +29,22 @@ def _assert_authorized_flac(self, **kwargs):
         # email = self.token_email
         # group = self.token_group
         # Do FLAC lookup here
-        return
+        self.assert_required_parameters(kwargs, ["uuid", "method"])
+        uuid = kwargs.get('uuid')
+
+        try:
+            flac_attributes = db.get_item(table=self.ddb_table_name, hash_key=uuid)
+        except db.DynamoDBItemNotFound as ex:
+            msg = f'uuid: {uuid} was not found in the flac table'
+            logger.info(msg, ex)
+            return
+        else:
+            if not self.assert_auth0authz_groups_intersects(flac_attributes['groups']):
```

Comment:
Hmm... if the method name starts with `assert_*` I would expect we would be able to just call it, and it would raise the exception. We should either have `assert_auth0authz_groups_intersects` raise the assertion, or rename it to something like `check_auth0authz_group_intersects`

Reply:
That's my bad! Lemme make a PR to fix that method real quick

Reply:
See #152